src/pudl_rmi/coordinate.py

```diff
@@ -0,0 +1,247 @@
+                False.
+        """
+        if not pudl_rmi.FILE_PATH_PLANT_PARTS_EIA.is_file() or clobber:
+            logger.info(
+                "Master unit list not found "
+                f"{pudl_rmi.FILE_PATH_PLANT_PARTS_EIA} Generating a new master"
+                " unit list. This should take ~10 minutes."
+            )
+            # actually make the master plant parts list
+            plant_parts_eia = self.pudl_out.plant_parts_eia()
+            # export
+            plant_parts_eia.to_pickle(
+                pudl_rmi.FILE_PATH_PLANT_PARTS_EIA, compression='gzip'
+            )
+        else:
+            logger.info(
+                "Reading the plant part list from "
+                f"{pudl_rmi.FILE_PATH_PLANT_PARTS_EIA}"
+            )
+            plant_parts_eia = pd.read_pickle(
+                pudl_rmi.FILE_PATH_PLANT_PARTS_EIA, compression='gzip'
+            )
+        return plant_parts_eia
+
+    def get_deprish(self, clobber=False):
+        """
+        Generate of grab the cleaned deprecaition studies.
+
+        Args:
+            clobber (boolean): True if you want to regenerate the depreciation
+                data whether or not the output is already pickled. Default is
+                False.
+        """
+        if not pudl_rmi.FILE_PATH_DEPRISH.is_file() or clobber:
+            logger.info("Generating new depreciation study output.")
+            deprish_df = pudl_rmi.deprish.execute()
+            deprish_df.to_pickle(pudl_rmi.FILE_PATH_DEPRISH)
+        else:
+            logger.info(
+                "Grabbing depreciation study output from "
+                f"{pudl_rmi.FILE_PATH_DEPRISH}"
+            )
+            deprish_df = pd.read_pickle(pudl_rmi.FILE_PATH_DEPRISH)
+        return deprish_df
+
+    def get_deprish_to_eia(
+        self,
+        clobber=False,
+        clobber_ppl=False,
+        clobber_d=False
+    ):
+        """
+        Generate or grab the connection between the depreciation data and EIA.
+
+        Args:
+            clobber (boolean): True if you want to regenerate the connection
+                between the depreciation data and EIA whether or not the output
+                is already pickled. Default is False.
+            clobber_ppl (boolean): True if you want to regenerate the master
+                unit list whether or not the output is already pickled. Default
+                is False.
+            clobber_d (boolean): True if you want to regenerate the
+                depreciation data whether or not the output is already pickled.
+                Default is False.
+        """
+        clobber_any = any([clobber, clobber_ppl, clobber_d])
+        if not pudl_rmi.FILE_PATH_DEPRISH_EIA.is_file() or clobber_any:
+            deprish_match_df = pudl_rmi.connect_deprish_to_eia.execute(
+                plant_parts_df=self.get_plant_part_list(clobber=clobber_ppl),
+                # deprish_df=self.get_deprish(clobber=clobber_d),
```

Comment:
Is this supposed to be commented out? It seems like it's not possible to clobber the depreciation file as it is.

Reply:
I should remove this.